Formula/p/python-freethreading.rb

```diff
@@ -279,8 +282,10 @@ def install
     mv bin/"idle#{version.major_minor}", bin/"idle#{version.major_minor}t"
     mv bin/"pydoc#{version.major_minor}", bin/"pydoc#{version.major_minor}t"
 
-    # Remove python3.13 named executables
-    (bin.children - bin.glob("*#{version.major_minor}t*")).map(&:unlink)
+    # Remove files that conflict with the main python3 formula
+    [bin, lib, lib/"pkgconfig", include].each do |directory|
+      (directory.glob("*python*") - directory.glob("*#{version.major_minor}t*")).map(&:unlink)
```

Comment:
Actually, `idle3` and `pydoc3` will not be removed after this change, both of which still conflict with `python@3.13`.

Reply:
Thanks for checking! It seems that `idle3` and `pydoc3` have already been handled, but `pip3` seems to not have been:
```
❯ ls -la $(brew --prefix python-freethreading)/bin
total 56
drwxr-xr-- 2 linuxbrew linuxbrew  4096 Nov  2 14:25 .
drwxr-xr-- 7 linuxbrew linuxbrew  4096 Nov  2 14:25 ..
-r-xr-xr-- 1 linuxbrew linuxbrew   152 Nov  2 14:25 idle3.13t
-rwxr-xrwx 1 linuxbrew linuxbrew   272 Nov  2 14:25 pip3
-rwxr-xrwx 1 linuxbrew linuxbrew   272 Nov  2 14:25 pip3.13t
-r-xr-xr-- 1 linuxbrew linuxbrew   137 Nov  2 14:25 pydoc3.13t
-r-xr-xr-- 1 linuxbrew linuxbrew 22744 Nov  2 14:25 python3.13t
-r-xr-xr-- 1 linuxbrew linuxbrew  3198 Nov  2 14:25 python3.13t-config
-rwxr-xrwx 1 linuxbrew linuxbrew   259 Nov  2 14:25 wheel3.13t
```